test/addurl_test.go

```diff
@@ -69,8 +69,10 @@ TAG ` + tag
 	})
 	defer os.RemoveAll(buildDir)
 
-	err := runRockerBuildWdWithOptions(buildDir, "-cache-dir", cacheDir)
-	assert.Nil(t, err, "no difference in downloaded and added files")
+	err := runRockerBuildWd(buildDir, "-cache-dir", cacheDir)
+	if err != nil {
```

Comment:
Note that you will also have to inverse the value like this, which is not as beautiful:
```go
if !assert.NoError(t, err) {
...
```